src/service/url-replacements-impl.js

```diff
@@ -213,8 +221,22 @@ export class UrlReplacements {
           scope: dev().assertString(scope),
           createCookieIfNotPresent: true,
         }, consent);
+      }).then(cid => {
+        if (!clientIds) {
+          clientIds = Object.create(null);
+        }
+        clientIds[scope] = cid;
+        return cid;
       });
     });
+    // Synchronous alternative. Only works for scopes that were previously
+    // requested using the async method.
+    this.set_('CLIENT_ID', scope => {
+      if (!clientIds) {
+        return null;
+      }
+      return clientIds[dev().assertString(scope)];
```

Comment:
don't quite understand what's the scope here. Is it always the same as what we set `clientIds[scope` above? 
Would it be possible to get`clientIds[scope] = null` here?